pkg/kubectl/cmd/drain.go

```diff
@@ -184,6 +185,7 @@ func NewCmdDrain(f cmdutil.Factory, out, errOut io.Writer) *cobra.Command {
 		},
 	}
 	cmd.Flags().BoolVar(&options.Force, "force", false, "Continue even if there are pods not managed by a ReplicationController, ReplicaSet, Job, DaemonSet or StatefulSet.")
+	cmd.Flags().BoolVar(&options.IgnoreNotFound, "ignore-not-found", false, "Ignore \"resource not found\" error when no matching controller is found.")
```

Comment:
Suppose there is a pod managed by Daemonsets controller but the controller is gone for any reason. What if users specify `ignore-not-found` and `ignore-daemonsets` ?